src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -342,9 +348,10 @@ protected function resetMigrations(array $migrations, array $paths, $pretend = f
      * @param  string  $file
      * @param  object  $migration
      * @param  bool  $pretend
+     * @param  bool  $fake
      * @return void
      */
-    protected function runDown($file, $migration, $pretend)
+    protected function runDown($file, $migration, $pretend, $fake)
```

Comment:
This is a breaking change unfortunately.

Reply:
```suggestion
    protected function runDown($file, $migration, $pretend, $fake = false)
```